### ISSUE TITLE
German ELECTRA?

### ISSUE THREAD

**Issue by lucacampanella** [dbmdz/berts#22]
Hi guys, tremendous work, I love using your german pretrained models. I was wondering if there is any plan to release a pretrained version of ELECTRA. Its seems like a really good approach.
Thanks!
Luca :)

**Comment by lutz-100worte** [comment]
Check out "German's Next Language Model" in the main track of [COLING 2020](https://coling2020.org/pages/accepted_papers_main_conference):

> In this work we present the experiments which lead to the creation of our BERT and ELECTRA based German language models, GBERT and GELECTRA. By varying the input training data, model size, and the presence of Whole Word Masking (WWM) we were able to attain SoTA performance across a set of document classification and named entity recognition (NER) tasks for both models of base and large size. We adopt an evaluation driven approach in training these models and our results indicate that both adding more data and utilizing WWM improve model performance. By benchmarking against existing German models, we show that these models are the best German models to date. All trained models will be made publicly available to the research community.

**Comment by Timoeller** [comment]
Thanks for the praise. And very good timing. We are just in the finalization for releasing the models mentioned in our coling paper on the model hub. Today or latest tomorrow they will be up and running. See progress in deepset-ai/FARM/issues/567

**Comment by lucacampanella** [comment]
Thanks to both of you for your quick replies and your work. Good timing indeed. I'll wait for the release :)

**Comment by stefan-it** [comment]
Hi @lucacampanella ,

the models were released quite a while ago, so I'm closing that issue here - have fun with the models :hugs:
